core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -65,6 +65,12 @@ abstract class SparkFunSuite
   with Logging {
 // scalastyle:on
 
+  // Initialize the logger forcibly to let the logger log timestamp
+  // based on the local time zone depending on environments.
+  // The default time zone will be set to America/Los_Angeles later
+  // so this initialization is necessary here.
+  log
```

Comment:
hm, `TimeZone.setDefault` below doesn't an affect the already-instanciated logger?

Reply:
Seems so. `DateFormat` classes used in log4j uses `Calendar`, which is instantiated in the constructor with the default time zone at the time.

Reply:
okay, if so, the change looks reasonable. cc: @srowen